autotest/pst_from_tests.py

```
@@ -1279,15 +1272,12 @@ def mf6_freyberg_shortnames_test(tmp_path):
 
 
 def mf6_freyberg_da_test(tmp_path):
+    flopy = pytest.importorskip("flopy")
```

I think for these we can just do a straight import flopy. flopy should be defined as a testing dependency (only) so if you are running these tests you do need flopy so these would be better to raise rather than skip?